env.py

```diff
@@ -100,99 +122,125 @@ def issue_survey():
             minimum_docker_version = vp("19.03.12")
             if container_version < minimum_docker_version:
                 container_message = "BELOW MINIMUM: UPDATE DOCKER"
-                
-        final_report += textwrap.dedent("""\
+
+        final_report += textwrap.dedent(
+            """\
             Container Engine: %s v%s (%s)
-        """ % (os_info.container_info.engine, container_version, container_message))
+        """
+            % (os_info.container_info.engine, container_version, container_message)
+        )
     else:
         try:
-            subprocess.check_output(["systemd-detect-virt", "-c"]) # Check if running inside container
+            subprocess.check_output(
+                ["systemd-detect-virt", "-c"]
+            )  # Check if running inside container
             if not os.path.exists("/openlane"):
                 raise Exception()
             alert = "Alert: You appear to be running this script inside the OpenLane container. This is not the intended usecase for this file."
-            final_report += ("%s\n" % alert)
+            final_report += "%s\n" % alert
             print(alert, file=alerts)
-        except Exception as e:
-            alert = "Critical Alert: No Docker or Docker-compatible container engine was found."
-            final_report += ("%s\n" % alert)
+        except Exception:
+            alert = "Critical Alert: No Docker or Docker-compatible container engine was found: {e}."
```

Comment:
also I think you didn't mean to put `{e}` as this is not and f string

Reply:
yeah thanks